docs/integrating-into-frameworks.md

```diff
@@ -14,11 +14,10 @@ frameworks.
 
 ## Examples
 
-Our [framework-examples](../framework-examples) directory contains sample applications built in various frameworks,
-such as [react](../framework-examples/react) and [angular2](../framework-examples/angular2) that show how to wrap our
-components. If there's an example framework you'd like to see, or if you're a framework author who
-wants to provide an example, please consider letting us know and we'll work with you to have it
-contributed!
+We maintain a list of component libraries, which wrap MDC Web for another framework, on our main [README](../README.md). Each library must:
```

Comment:
"for other frameworks"

Too me that makes it clear that we're not looking for one particular framework that may or may not be developed by Google.